Sources/TestUtils/MockKeyValueStore.swift

```diff
@@ -19,11 +19,11 @@
 import Foundation
 import Persistence
 
-public class MockKeyValueStore: KeyValueStoring {
+public class MockKeyValueStore: NSObject, KeyValueStoring {
```

Comment:
We can undo the changes in this file since they were only added when a KVO expectation was being used in the test suite. Now that it's not, we can scrap the change.